### PR TITLE
[WIP] Add get bdd2

### PR DESCRIPTION
**Pull request by lombardero**
Started adding `GET` checks

### CODE REVIEW

File: service/service.py [hunk 1]
```diff
@@ -148,6 +148,17 @@ def delete_wishlists(wishlist_id):
         wishlist.delete()
     return make_response('', status.HTTP_204_NO_CONTENT)
 
+######################################################################
+# DELETE ALL PET DATA (for testing only)
+######################################################################
+@app.route('/wishlists/reset', methods=['DELETE'])
```
Comment (nisargthakkar) [inline service/service.py:154]: Since we're not using the endpoint, can we remove it?

File: service/models.py [hunk 2]
```diff
@@ -157,6 +157,15 @@ def find_by_all(cls, wishlist_id=None, name=None, customer_id=None):
 
         return cls.query.filter(*queries)
 
+    @classmethod
+    def remove_all(cls):
```
Comment (nisargthakkar) [inline service/models.py:161]: Since we're not using the endpoint, can we remove it?

File: features/wishlist.feature [hunk 3]
```diff
@@ -102,6 +107,13 @@ Scenario: Update a Wishlist
     And I press the "Search" button
     Then I should see Name = "before today", Customer ID = "128" in the results
 
+Scenario: List all Wishlists
```
Comment (nisargthakkar) [inline features/wishlist.feature:110]: Please make this in a separate PR as it is linked to a different story

File: features/wishlist.feature [hunk 4]
```diff
@@ -10,6 +10,11 @@ Background:
         | 256         | today          |
         | 512         | tomorrow       |
 
+Scenario: The server is running
+    When I visit the "Home Page"
```
Comment (nisargthakkar) [inline features/wishlist.feature:14]: Can you also add a `Given` clause?

File: features/steps/wishlist_steps.py [hunk 5]
```diff
@@ -68,6 +76,8 @@ def step_impl(context):
 def step_impl(context):
     """ Delete all Wishlists and load new ones """
     headers = {'Content-Type': 'application/json'}
+    # context.resp = requests.delete(context.base_url + '/wishlists/reset', headers=headers)
```
Comment (nisargthakkar) [inline features/steps/wishlist_steps.py:79]: Can you remove this commented code?

File: tests/test_service.py [hunk 6]
```diff
@@ -685,6 +685,25 @@ def test_rename_wishlist_product_invalid_content_type(self):
                                                             'product_name': 'surface_pro'},
                                                         headers={'content-type': 'text/plain'})
         self.assertEqual(resp.status_code, status.HTTP_415_UNSUPPORTED_MEDIA_TYPE)
+    
+    def test_remove_all_wishlist(self):
```
Comment (nisargthakkar) [inline tests/test_service.py:689]: Since we're not using the endpoint, can we remove the tests as well?

**Review by nisargthakkar — CHANGES_REQUESTED** [review]

File: features/wishlist.feature (hunk 4 above)
Comment (lombardero) [reply features/wishlist.feature:14]: No need to add anything, the `given` clause is the `Given the following wishlists`.

File: features/steps/wishlist_steps.py (hunk 5 above)
Comment (lombardero) [reply features/steps/wishlist_steps.py:79]: It is there since I will try to make it work. I added this PR so you could work, I can undo the PR if you do not want this commented code.

File: tests/test_service.py (hunk 6 above)
Comment (lombardero) [reply tests/test_service.py:689]: Same as before, I want to try to make it work. Can we let it be for now?

File: service/service.py (hunk 1 above)
Comment (lombardero) [reply service/service.py:154]: No, I want to still use it. I can cancel the PR if that is better since tomorrow I will work on it.

File: features/wishlist.feature (hunk 3 above)
Comment (lombardero) [reply features/wishlist.feature:110]: This precisely testing the `GET` request, the story I am working on. I do not get why we need a different story.

If you want, I can add in a different PR the `Scenario: Server is running`

File: service/models.py (hunk 2 above)
Comment (lombardero) [reply service/models.py:161]: Se above comment

File: tests/test_service.py (hunk 6 above)
Comment (nisargthakkar) [reply tests/test_service.py:689]: Oh ok. No worries. I thought this was the final one to be merged. My changes are already merged so I'm not blocked due to this anymore. Feel free to make all changes as you wish

File: service/service.py (hunk 1 above)
Comment (nisargthakkar) [reply service/service.py:154]: Not needed. We can let it be open. You can put a title saying do not merge or work in progress or something

File: features/wishlist.feature (hunk 3 above)
Comment (nisargthakkar) [reply features/wishlist.feature:110]: There were two stories for two endpoints. Let it be. We'll delete the get index story
